js/widgets/temperament.js

```diff
@@ -2169,11 +2157,68 @@ class TemperamentWidget {
                 i -= 1;
             }
 
-            if (i <= pitchNumber && i >= 0 && this._playing && p < 2) {
+            if (i <= pitchNumber && i >= 0 && p < 2) {
                 setTimeout(() => {
-                    __playLoop(i);
+                    if (this.closed) {
+                        this.closed = !this.closed;
+                        return;
+                    } else if (!this._playing) {
+                        cell.innerHTML =
+                            '&nbsp;&nbsp;<img src="header-icons/' +
+                            "play-button.svg" +
+                            '" title="' +
+                            _("Play") +
+                            '" alt="' +
+                            _("Play") +
+                            '" height="' +
+                            TemperamentWidget.ICONSIZE +
+                            '" width="' +
+                            TemperamentWidget.ICONSIZE +
+                            '" vertical-align="middle" align-content="center">&nbsp;&nbsp;';
+                        if (i !== -1) {
+                            if (this.circleIsVisible == false && docById("wheelDiv4") == null) {
+                                this.notesCircle.navItems[i - 1].fillAttr = "#c8C8C8";
+                                this.notesCircle.navItems[i - 1].sliceHoverAttr.fill = "#c8C8C8";
```

Comment:
c8C8C8 should be a constant

Reply:
But, how will it be beneficial?

Reply:
One advantage of using a constant symbol instead of a magic number/string is that you can express the semantics of the value more precisely.

Reply:
Done.